### ISSUE TITLE
Setup issues running openjdk tests on multi-core machines 

### ISSUE THREAD

**Issue by smlambert** [AdoptOpenJDK/openjdk-tests#1203]
Applies to openjdk / jtreg tests, may be related to https://bugs.openjdk.java.net/browse/CODETOOLS-7902454

We should confirm other test groups work (system, perf, functional).

Seen on `test-packet-armv8-ubuntu-16-04`

When tests run on that box they usually fail in some weird way, like:
```#
00:24:50  # There is insufficient memory for the Java Runtime Environment to continue.
00:24:50  # Native memory allocation (mmap) failed to map 526385152 bytes for committing reserved memory.
00:24:50  # An error report file with more information is saved as:```

Perhaps some problem running on massive multicore machines (96 cores) vs. the lower-end machines (8 cores) where all seems to work OK.

**Comment by jerboaa** [comment]
FWIW #1427 will fix this, once merged. The actual issue is a `-concurrency=$(nproc)` setting for JTREG. I.e. it currently sets concurrency to the total number of procesors (or cores).

**Comment by jerboaa** [comment]
Fixed with #1427
